### ISSUE TITLE
Group 1 changes

### ISSUE THREAD

**Issue by kusala9** [iho-ohi/S-57-to-S-101-conversion-sub-WG#1]
How to deal with the Group 1 changes in S-101. Placeholder.

**Comment by kusala9** [comment]
![image](https://user-images.githubusercontent.com/3368156/103462452-2a0fc900-4d1d-11eb-95fe-fe170d67909f.png)

group 1 changes illustrated above

**Comment by kusala9** [comment]
this diag was in the first report we did on conversion and shows the changes between S-57 and S-101. The issue can be broken down as follows:
* Features which are in both S-101 and S-57 can be left as is. No need for changes (DEPARE, DRGARE, LNDARE, UNSARE
* Features which were Group 1 in S-57 and aren't in S-101 need to be replaced (probably) by a feature coincident with them which is in group 1 in S-101... So, to convert a HULKES feature to S-101, create, for instance an UnsurveyedArea coincident with the Hulkes and covnert the HULKES to its equivalent, "Hulk"? 
* Features which were NOT group 1 in S-57 but are in S-101. These need to be "cut into" the S-101 group 1 features. Easiest if they could be made coincident with a group 1 feature in S-57 which can then be swopped out during the conversion. So, a DOCARE in S-57 should be coincident with its underlying group 1 feature, then during conversion the DOCARE can be made the group 1 feature.

there's an implicit "choice" here to be made by the data producer in deciding which group 1 features to replace with, and whether to substitute (in the third case above). This could also benefit from some examples to see if it provides an automatable method.

**Comment by kusala9** [comment]
![image](https://user-images.githubusercontent.com/3368156/103462579-1add4b00-4d1e-11eb-8c7a-f87b10c75705.png)

Also from the original report. This is the HULKES example from the previous comment which may illustrate the second case above. A diagram of the third case would be useful...

**Comment by Christian-Shom** [comment]
- **HULKES**: **agree that Unsurveyed Area should be created during the conversion** as S-101 SOE (Skin Of the Earth) coincident with Hulk (this would need no change to prepare the S-57 data). Ideally there should be Depth Areas, but (recent) bathymetry under the hulk may be unknown (there may be obstructions), and the hulk may overlap different depth areas which complicates the conversion.
- **FLODOC and PONTON**: as these objects are also situated in the water, **in favour of having the same rule then above** for these objects.
- **DOCARE and LOKBSN**: S-57 underneath SOE object (LNDARE, UNSARE, etc.) to be "cut off". **Probably needs to be tested if geometry have to be separated first by the HO or if the converters can do the job.**

**Comment by kusala9** [comment]
If the DOCARE/LOKBSN have a coincident polygon shared with and existing group1 (e.g. DEPARE) then it's much easier as a new feature can be created. Otherwise some geometry intersection and topology operations have to happen in the converter (difficult!). so, this might be one where it can be made easier by pre-preparing the conversion process...

**Comment by tdepuyt** [comment]
I agree with maintaining the decision to put an Unsurveyed Area under HULKES/FLODOC/PONTON.   This would be the simplest approach for initial conversion.    There could be ways to prepare depth area information ahead of time but those conversations might end up being production platform specific.

Since we aren't swapping one feature for another as with the HULKES/FLODOC/PONTON example, DOCARE and LOKBSN will require new geometries to be created when holes are punched into existing G1 features.   This is more complex but possible to do during conversion.  5 years ago (S-100WG5) the group wasn't comfortable with the fact that this would alter existing geometries but we are already doing that with other scenarios so that shouldn't be a roadblock.   I'm also open to "pre-preparing" the conversion process as well but as stated above those could end up being specific to the production platform.

**Comment by JeffWootton** [comment]
Taking into consideration the possible requirement to support the reverse conversion (S-101 to S-57), I support the inclusion of Unsurveyed Area under the items in question.
Whether the conversion guidance recommends pre-preparation by amending the geometry or supports the geometry changes to be implemented by the converter depends on the capability of the converter.  An indication of this capability will be supplied through testing, after which the appropriate guidance can be included in the document.

**Comment by Christian-Shom** [comment]
Due to the removal of Dock Area and Lock Basin from Skin of the Earth in S-101, there are only two cases left:

- S-57 LNDARE, DEPARE, DGRARE and UNSARE whose equivalents in S-101 are also SOE features --> no issue for conversion (1 to 1);
- S-57 HULKES, PONTON and FLODOC whose equivalent in S-101 are not SOE features --> I suggest changing the guidance in the conversion document 1.2.0 as follows (example for FLODOC, to be copied for HULKES and PONTON):

![image](https://github.com/iho-ohi/S-57-to-S-101-conversion-sub-WG/assets/71139640/b86896eb-6aac-4a40-a540-1d5dd4522296)

If a CTNARE has been encoded to indicate the temporality/periodicity of a HULKES, PONTON or FLODOC, automization to populate S-101 temporal attributes seems difficult to achieve. I suggest leaving the conversion guidance as it is.

**Comment by JeffWootton** [comment]
Suggest that this Issue is closed. This was addressed for S-65 Annex B, Edition 1.0.0, and there has been no adverse feedback.
